### PR TITLE
SR Updates Tab

### PR DESCRIPTION
**Pull request by midhun-aot**

### CODE REVIEW

File: backend/sites/src/app/resolvers/associatedSite/associatedSite.resolver.ts [hunk 1]
```diff
@@ -34,14 +34,17 @@ export class AssociatedSiteResolver {
   @UsePipes(new GenericValidationPipe()) // Apply generic validation pipe
   async getAssociatedSitesBySiteId(
     @Args('siteId', { type: () => String }) siteId: string,
+    @Args('pending', { type: () => Boolean, nullable: true })
+    showPending: boolean,
+    
   ) {
     this.sitesLogger.log(
       'AssociatedSiteResolver.getAssociatedSitesBySiteId() start siteID:' +
```
Comment (nikhila-aot) [inline backend/sites/src/app/resolvers/associatedSite/associatedSite.resolver.ts:42]: please have the new parameter captured in the logs

File: backend/sites/src/app/resolvers/cart/cart.resolver.ts [hunk 2]
```diff
@@ -51,7 +51,7 @@ export class CartResolver {
         this.sitesLogger.log('CartResolver.getCartItemsForUser() RES:422 end');
```
Comment (nikhila-aot) [inline backend/sites/src/app/resolvers/cart/cart.resolver.ts]: please correct the response code in the logs

File: backend/sites/src/app/resolvers/disclosure/disclosure.resolver.ts [hunk 3]
```diff
@@ -31,14 +31,16 @@ export class DisclosureResolver {
   @UsePipes(new GenericValidationPipe()) // Apply generic validation pipe
   async getSiteDisclosureBySiteId(
     @Args('siteId', { type: () => String }) siteId: string,
+    @Args('pending', { type: () => Boolean, nullable: true })
+    showPending: boolean,
   ) {
     this.sitesLogger.log(
       'DisclosureResolver.getSiteDisclosureBySiteId() start siteId:' +
         ' ' +
```
Comment (nikhila-aot) [inline backend/sites/src/app/resolvers/disclosure/disclosure.resolver.ts:39]: capture the new param in the logs

File: backend/sites/src/app/resolvers/document/document.resolver.ts [hunk 4]
```diff
@@ -31,16 +33,22 @@ export class DocumentResolver {
...
   async getSiteDocumentsBySiteId(
     @Args('siteId', { type: () => String }) siteId: string,
+    @Args('pending', { type: () => Boolean, nullable: true })
+    showPending: boolean,
   ) {
-    sitesLogger.info(
+
+    this.sitesLogger.log(
       'DocumentResolver.getSiteDocumentsBySiteId() start siteId:' +
         ' ' +
```
Comment (nikhila-aot) [inline backend/sites/src/app/resolvers/document/document.resolver.ts:42]: capture the new param in the logs

File: backend/sites/src/app/resolvers/notation/notation.resolver.ts [hunk 5]
```diff
@@ -31,11 +31,17 @@ export class NotationResolver {
   @UsePipes(new GenericValidationPipe()) // Apply generic validation pipe
   async getSiteNotationBySiteId(
     @Args('siteId', { type: () => String }) siteId: string,
+    @Args('pending', { type: () => Boolean, nullable: true })
+    showPending: boolean,
   ) {
     this.sitesLogger.log(
       'NotationResolver.getSiteNotationBySiteId() start siteId:' + ' ' + siteId,
```
Comment (nikhila-aot) [inline backend/sites/src/app/resolvers/notation/notation.resolver.ts]: capture the new param in the logs

File: backend/sites/src/app/resolvers/participant/participant.resolver.ts [hunk 6]
```diff
@@ -30,14 +30,16 @@ export class ParticipantResolver {
   @UsePipes(new GenericValidationPipe()) // Apply generic validation pipe
   async getSiteParticipantsBySiteId(
     @Args('siteId', { type: () => String }) siteId: string,
+    @Args('pending', { type: () => Boolean, nullable: true })
+    showPending: boolean,
   ) {
     this.sitesLogger.log(
       'ParticipantResolver.getSiteParticipantsBySiteId() start siteId:' +
         ' ' +
```
Comment (nikhila-aot) [inline backend/sites/src/app/resolvers/participant/participant.resolver.ts:38]: capture the new param in the logs

File: backend/sites/src/app/resolvers/site/site.resolver.ts [hunk 7]
```diff
@@ -138,11 +138,14 @@ export class SiteResolver {
     mode: RoleMatchingMode.ANY,
   })
   @Query(() => FetchSiteDetail, { name: 'findSiteBySiteId' })
-  findSiteBySiteId(@Args('siteId', { type: () => String }) siteId: string) {
+  findSiteBySiteId(@Args('siteId', { type: () => String }) siteId: string,
+  @Args('pending', { type: () => Boolean, nullable: true })
+  showPending: boolean,
+) {
     this.sitesLogger.log(
       'SiteResolver.findSiteBySiteId() start siteId:' + ' ' + siteId,
```
Comment (nikhila-aot) [inline backend/sites/src/app/resolvers/site/site.resolver.ts]: capture the new param in the logs

**Review by nikhila-aot — CHANGES_REQUESTED** [review]

File: backend/sites/src/app/resolvers/associatedSite/associatedSite.resolver.ts (hunk 1 above)
Comment (midhun-aot) [reply backend/sites/src/app/resolvers/associatedSite/associatedSite.resolver.ts:42]: done

File: backend/sites/src/app/resolvers/disclosure/disclosure.resolver.ts (hunk 3 above)
Comment (midhun-aot) [reply backend/sites/src/app/resolvers/disclosure/disclosure.resolver.ts:39]: Done

File: backend/sites/src/app/resolvers/document/document.resolver.ts (hunk 4 above)
Comment (midhun-aot) [reply backend/sites/src/app/resolvers/document/document.resolver.ts:42]: done

File: backend/sites/src/app/resolvers/notation/notation.resolver.ts (hunk 5 above)
Comment (midhun-aot) [reply backend/sites/src/app/resolvers/notation/notation.resolver.ts]: done

File: backend/sites/src/app/resolvers/participant/participant.resolver.ts (hunk 6 above)
Comment (midhun-aot) [reply backend/sites/src/app/resolvers/participant/participant.resolver.ts:38]: Done

File: backend/sites/src/app/resolvers/site/site.resolver.ts (hunk 7 above)
Comment (midhun-aot) [reply backend/sites/src/app/resolvers/site/site.resolver.ts]: Done

File: backend/sites/src/app/resolvers/cart/cart.resolver.ts (hunk 2 above)
Comment (midhun-aot) [reply backend/sites/src/app/resolvers/cart/cart.resolver.ts]: Done

**Review by nikhila-aot — APPROVED** [review]

**Pull request merged**